js/utils/draggable_background.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Draggable Background plugin for jQuery
+ *
+ * v1.2.4
+ *
+ * Copyright (c) 2014 Kenneth Chung
+ *
+ * Licensed under the MIT license:
+ *   http://www.opensource.org/licenses/mit-license.php
+ */
+
+/* eslint-disable */
```

Comment:
I'm assuming this isn't available via npm?

Anyhow, for 3rd party code that we need to have in our code base (e.g. not npm-able), I have a js/lib folder for that, which I believe is set so the entire folder is ignored by eslint.